### ISSUE TITLE
Prodental - fichier source

### ISSUE THREAD

**Issue by nicolasdb** [openfab-lab/devis-et-facturation#3]
`Title = nom du projet + client`

#### Contexte :
Le fichier source svg est en plein du jeu de quilles sur ce repo. 

#### Objectif :
Est-ce le bon endroit pour stocker les fichiers sources des jobs? 
Ce n'est pas facile pour le récupérer, pas de lien de download, c'est pas clair. La solution par défaut est de faire un clique droit et `save img as...` mais le nom n'est pas gardé, ce qui n'est pas grave mais peut déjà être source de confusion par la suite. 
#### Process :
Si on garde une logique github, on pourrait avoir un repo privé pour les fichiers de travail. 
Mais ce que je préférerai c'est de garder une logique de backup privé. Comme aujourd'hui, les fichiers job sont dans un dossier job sur le NAS, et synchronisé avec mon ordi et un autre ici au bureau, avec [syncthing](https://syncthing.net/) (une solution dropbox peer2peer, sécurisée, non-cloud et opensource)

ça demande une petite configuration pour identifier l'ordi mais une fois que c'est fait, on se synchronise sur le NAS dès qu'on est en ligne = solution à distance. 

autre solution non opensource, cloud based, c'est le drive. Milena l'utilise avec l'app desktop qui s'occupe de cette synchronisation aussi.
Faudrai faire un gros travail de nettoyage et d'organisation du (des) drives, j'avoue que c'est tjs un peu le bordel chez moi avec tout les comptes google différents. 

---

Qu'en pensez vous?

**Comment by nicolasdb** [comment]
it's time to go non profit right? 
https://github.com/nonprofit

**Comment by zuperninja** [comment]
Sinon faire un dossier 2print :)

**Comment by satblip** [comment]
@nicolasdb tu veux que je m'en charge ?

**Comment by nicolasdb** [comment]
Yup, Louis, si ça te va, ça serai super. 

Un fichier 2print y en a déjà un sur le NAS mais c'est pas ce qu'il faut. C'est surtout un moyen de centraliser les fichiers de job clients (+ backup) 
On peut pas laisser ça publique, et surtout, on doit bien les ranger quelque part pour futures références. 
\+ note! Certains de ces fichiers peuvent être volumineux et ce n'est pas l'usage de github.

**Comment by satblip** [comment]
Alors j'ai fait la demande, on attend de leur nouvelles (surement semaine prochaine). J'ai remplis aussi pour Google Nonprofit, pareil, on attend de leurs nouvelles ;)

**Comment by satblip** [comment]
@nicolasdb @zuperninja btw, pour faciliter un peu l'échange avec le NAS, on bougerait pas tout sur google drive ? SI on a le compte nonprofit, on aura de l'espace de stockage.

**Comment by zuperninja** [comment]
Yes ça peut être l'occase de trier. Nas en back up/archives et le reste sur google drive ?

**Comment by nicolasdb** [comment]
Ouais, 
- on peut avoir l'admin sur le drive avec une copie en local sur le NAS. 
- on peut avoir des dossiers de travails en cours sur le drive. 
- et les archives sur le NAS. 
Mais il va falloir des comptes pour tous et une 
![image](https://user-images.githubusercontent.com/12049360/46257824-e3d5ba80-c4c0-11e8-845a-85e58858356e.png) 
et un respect de l'ordre digne du Smithsonian

**Comment by nicolasdb** [comment]
![image](https://user-images.githubusercontent.com/12049360/46257874-c2290300-c4c1-11e8-9306-d4a305ba18fb.png)


https://www.slideshare.net/joelrichard/linked-open-data-and-systematic-taxonomy

**Comment by satblip** [comment]
`Mais il va falloir des comptes pour tous`  pas spécialement, on peut share des folders par job, sans qu'ils aient un compte syntonie (pcq bonjour l'admin :-D)

**Comment by zuperninja** [comment]
Oui nomenclature quoi !!!
Compte par job ça me plait :)

**Comment by satblip** [comment]
folder par job, pas compte :D On fait un folder et on le share avec les emails persos des personnes travaillants sur le job ;)

**Comment by nicolasdb** [comment]
![image](https://user-images.githubusercontent.com/12049360/46344613-06431180-c642-11e8-96b9-1b7063c120dc.png)

ça sent le gif ça!

**Comment by nicolasdb** [comment]
Bon, On a 30Go de drive. 
va falloir ranger le drive contact, y a du bordel, des photos en doubles et à la racine, des vieux documents vides ou complétement périmés. 

ET faire un petit guide de bonne pratique pour structurer le drive team. 
ET le mettre en anglais. Milena fait l'effort en français mais moi, je me sentirai perdu si on utilisait le néerlandais.

**Comment by nicolasdb** [comment]
yeah Alright. 
![gif](https://media.giphy.com/media/H100Qx5MfVmjC/giphy.gif) 


Anyway, I started to setup 2 drives and move some content as I need for accounting. 
So it's clean and clear. 

I've explained to AnSo aussi. 
But, could be neat to have a map to explain 
- what to use as platform, for what content. 
- when and of course how to use them. 
To avoid crossover and losing content into the mess
